docs/security-model.mdx

```diff
@@ -85,6 +85,12 @@ traffic (Kubernetes, OpenShift, ...).
 
 ## HTTP Cookies Explained
 
+:::tip
+
+Read our extensive blog post exploring [authentication and modern software](https://www.ory.sh/auth-and-modern-software/)
```

Comment:
I think the blog article is great. I think we should keep this section strictly about "HTTP Cookies Explained". What about adding this article to a "further/recommended reading" section at the end?

Reply:
I have been reading over the article a bit more and thought it might make sense to have this at the top right after introducing the topic. The reason being is that this docs article does not really explain all the differences between cookies and tokens or what a session really is. But we do mention that we manage it in the introductory paragraph. I can have the `tip` there.